vuepress/docs/next/docs/getting-started/multitenancy.md

```diff
@@ -0,0 +1,46 @@
+---
+sidebarDepth: 2
+---
+
+# Entando Multitenancy
+
+## Overview
+
+Entando 7.2 introduces multitenancy to the Entando Platform. An Entando Application can be customized to enable a multitenant architecture where tenants share an infrastructure but are securely informationally isolated. This document provides an overview of multitenancy and its Entando implementation.
+
+## Core Concepts
+
+Software multitenacy describes an architecture in which a single software instance runs on a server and serves multiple tenants. Each tenant constitutes a group of users who share specific access privileges to the software instance. A multitenant software application is designed to provide a common architecture while ensuring the segregation of information.
```

Comment:
first sentence: sp-multitenancy
Software multitenacy describes an architecture in which a single software instance runs on a server and serves multiple tenants.<< don't think you need both 'software'

Second sentence should maybe include something about its own unique domain name, and it makes it sound like it only applies to a private application when a tenant can be a public domain. `Access privileges to the software instance` can be read as access to that application in general, not just the access to engineer the software?